README.CentOS7.md

```diff
@@ -32,8 +32,8 @@ sudo yum install epel-release
 sudo yum install http://rpms.remirepo.net/enterprise/remi-release-7.rpm
 sudo yum install yum-utils
 sudo yum update
-sudo yum --enablerepo=remi-php72 install php php-xml php-mbstring
-sudo yum install php72-php-fpm php72-php-gd php72-php-json php72-php-mysqlnd php72-php-xmlrpc php72-php-opcache php72-php-pdo php72-php-mysql
+sudo yum --enablerepo=remi-php72 install php php-pdo php-pdo_mysql php-xml php-mbstring
+sudo yum install php72-php-fpm php72-php-gd php72-php-json php72-php-mbstring php72-php-mysqlnd php72-php-xml php72-php-xmlrpc php72-php-opcache php72-php-mysql
```

Comment:
```suggestion
sudo yum install php72-php-fpm php72-php-gd php72-php-json php72-php-mysqlnd php72-php-xmlrpc php72-php-opcache php72-php-mysql
```
packages installed on the previous line

Reply:
@PapillonMcGill committed your suggestion. Should I change it as your alternative suggestion?

Reply:
I believe so. It would be only 1 instruction and it would be more consistent.

Reply:
@PapillonMcGill I added it.